package.json

```diff
@@ -47,6 +47,7 @@
     "react-router-dom": "^5.0.1",
     "react-scripts": "^3.0.1",
     "redux": "^4.0.0",
+    "semver-compare": "^1.0.0",
```

Comment:
This lib is just a function with 13 lines. Maybe we should just copy it to our codebase instead of adding this dependency.

> Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

Reply:
Done.